OpenEdXMobile/src/main/java/org/edx/mobile/util/ConfigUtil.kt

```diff
@@ -27,6 +30,56 @@ class ConfigUtil {
             return false
         }
 
+        /**
+         * Utility method to check the Discovery status
+         *
+         * @param environment       Relevant configuration environment.
+         * @return True if Discovery enabled in configurations false else wise
+         */
+        fun isDiscoveryEnabled(environment: IEdxEnvironment? = null): Boolean {
+            return environment?.config?.discoveryConfig?.courseDiscoveryConfig?.isDiscoveryEnabled == true
+        }
+
+        /**
+         * Utility method to check the Degree Discovery status
+         *
+         * @param environment       Relevant configuration environment.
+         * @return True if Degree Discovery enabled in configurations false else wise
+         */
+        fun isDegreeDiscoveryEnabled(environment: IEdxEnvironment? = null): Boolean {
+            return environment?.config?.discoveryConfig?.degreeDiscoveryConfig?.isDiscoveryEnabled(environment) == true
+        }
+
+        /**
+         * Utility method to check the Webview Discovery status
+         *
+         * @param environment       Relevant configuration environment.
+         * @return True if Webview Discovery enabled in configurations false else wise
+         */
+        fun isWebviewDiscoveryEnabled(environment: IEdxEnvironment? = null): Boolean {
+            return environment?.config?.discoveryConfig?.courseDiscoveryConfig?.isWebviewDiscoveryEnabled == true
+        }
```

Comment:
IMO: this would be the better code.
```java
        /**
         * Utility method to check the Course Webview Discovery status
         *
         * @param environment       Relevant configuration environment.
         * @return true if Course Webview Discovery enabled in configurations false else wise
         */
        fun isCourseWebviewDiscoveryEnabled(environment: IEdxEnvironment): Boolean {
            return environment.config.discoveryConfig?.courseDiscoveryConfig?.isWebviewDiscoveryEnabled == true
        }
```